src/Conf.ml

```diff
@@ -305,7 +305,8 @@ end = struct
     let docs = section_name section in
     let term = Arg.(value & flag & info names_for_cmdline ~doc ~docs) in
     let parse s =
-      try Ok (Bool.of_string s) with _ ->
+      try Ok (Bool.of_string s)
+      with _ ->
```

Comment:
It helps quick understanding of the control flow to have `with` on the left, and doesn't cost indentation of the handler body.

Reply:
I agree it looks better